rules/composer/src/ValueObject/ComposerModifier/AddPackageToRequireDev.php

```diff
@@ -20,23 +19,18 @@ final class AddPackageToRequireDev implements ComposerModifierInterface
     private $packageName;
 
     /**
-     * @var Version
+     * @var string
      */
     private $version;
 
-    /**
-     * @param string $packageName name of package (vendor/package)
-     * @param string $version target package version (1.2.3, ^1.2, ~1.2.3 etc.)
-     */
     public function __construct(string $packageName, string $version)
     {
         $this->packageName = $packageName;
-        $this->version = new Version($version);
```

Comment:
now there is no validation, what about at least call `new Version($version);` at first line to validate version string?

Reply:
WIP :) validation should be in 1 place.